kbase-extension/static/kbase/js/widgets/kbaseTabs.js

```diff
@@ -2,54 +2,59 @@
 
     Easy widget to serve as a tabbed container.
 
+    It is an extension of the Bootstrap 3 tab (https://getbootstrap.com/docs/3.4/javascript/#tabs).
+
     var $tabs =  new kbaseTabs($('#tabs'), {
-            tabPosition : 'bottom', //or left or right or top. Defaults to 'top'
-            canDelete : true,       //whether or not the tab can be removed. Defaults to false.
+            canDelete : true,       // boolean; whether a tab which does not otherwise specify this option
+                                    //   will have a close button and can be removed. Defaults to false, overridden
+                                    //   if set on a tab.
```

Comment:
are these lines Prettier compliant? I can't remember if it reformats commented text or not. I ask because the lines look quite long and are wrapping a bit oddly in the GitHub display

Reply:
Prettier does not format comments, other than jsdoc ones. Yes, these are very wide, have been that way for probably over a decade.

Nevertheless, I extracted the code, moved the comments around fixed and modernized the code.